benchmarks/decoders/benchmark_decoders.py

```diff
@@ -198,6 +198,7 @@ def get_consecutive_frames_from_video(self, video_file, numFramesToDecode):
 
 
 def get_video_path(filename: str) -> str:
+    return os.path.join("/home/ahmads/personal/torchcodec/test/resources", filename)
```

Comment:
Debugging remnant. :)

Reply:
Good catch.  Actually I meant to fix this because this file is not part of the package -- __package__ is actually None since we run this from the source dir.